src/main/java/de/craften/ui/swingmaterial/Utils.java

```diff
@@ -0,0 +1,158 @@
+package de.craften.ui.swingmaterial;
+
+import java.awt.Color;
+import java.awt.Dimension;
+import java.awt.Frame;
+import java.awt.GraphicsConfiguration;
+import java.awt.GraphicsDevice;
+import java.awt.GraphicsEnvironment;
+import java.awt.Rectangle;
+import java.awt.Toolkit;
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import java.util.Arrays;
+
+/**
+ * Put miscellaneous utilitary static methods here.
+ * 
+ * @author DragShot
+ */
+public class Utils {
+    /**
+     * An integer flag for <code>getScreenSize()</code>.<br/><br/>
+     * Values:<br/>
+     *  0: Check if sun.java2d.SunGraphicsEnvironment.getUsableBounds() is
+     *     available.<br/>
+     *  1: Class/method is available.<br/>
+     * -1: Class/method is not available.
+     */
+    private static int useSun2D = 0;
```

Comment:
Meh, integer flags. This should be a `private static final boolean useSun2D` and we can check statically if it is available:

```java
private static final boolean useSun2D;
static {
  useSun2D = ...; // check
}
```